frontend/src/components/layout/Layout.tsx

```diff
@@ -6,44 +6,44 @@ import SideBar from "../sidebar/Sidebar";
 
 const useStyles = makeStyles((theme) => ({
   baseContainer: {
-    height: '100%'
+    height: "100%",
```

Comment:
my prettier made this code style changes. we should have a `prettier` config file to make it consistent across devs.

Reply:
definitely, we had some but we took it out because we didn't consider to have it for the mvp, due to time optimization of dev resources

Reply:
gotcha, PR #5 will take care of that